tests/pipelines/stable_diffusion_2/test_stable_diffusion_latent_upscale.py

```diff
@@ -173,11 +173,51 @@ def test_inference(self):
 
         self.assertEqual(image.shape, (1, 256, 256, 3))
         expected_slice = np.array(
-            [0.47222412, 0.41921633, 0.44717434, 0.46874192, 0.42588258, 0.46150726, 0.4677534, 0.45583832, 0.48579055]
+            [0.3970313, 0.3768756, 0.41147298, 0.4716793, 0.5115408, 0.44601366, 0.43763855, 0.46781355, 0.46358708]
```

Comment:
why do we need to update the expected_slice here? the results of existing tests should not change, no?